packages/fxa-settings/src/pages/SigninBounced/index.test.tsx

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { renderWithRouter } from '../../models/mocks';
+import { getFtlBundle, testL10n } from 'fxa-react/lib/test-utils';
+import { FluentBundle } from '@fluent/bundle';
+import SigninBounced from '.';
+import { logPageViewEvent } from '../../lib/metrics';
+
+jest.mock('../../lib/metrics', () => ({
+  logViewEvent: jest.fn(),
+  logPageViewEvent: jest.fn(),
+}));
+
+describe('SigninBounced', () => {
+  let bundle: FluentBundle;
+  let exampleEmail: string;
+  beforeAll(async () => {
+    bundle = await getFtlBundle('settings');
+    exampleEmail = 'example@domain.com';
+  });
+  it('renders default content as expected', () => {
+    renderWithRouter(<SigninBounced email={exampleEmail} />);
+    const ftlMsgMock = screen.getAllByTestId('ftlmsg-mock')[1];
+    testL10n(ftlMsgMock, bundle, {
+      email: exampleEmail,
+    });
+    // This is a little goofy, but I was really stumbling over how to parse for the unicode char.
+    const signinBouncedHeader = screen.getByText(/sorry/i);
```

Comment:
I played around with this a bit because I was curious if I could come up with an alternate solution.

A possible option might be switching the `&nbsp;` in `index` to `{'\u00A0'}`

and lines 32+34 here to:
```screen.getByRole('heading',{name:"Sorry. We’ve locked your\xa0account."});```

I've tested locally and this seems to pass.

Reply:
That is a great solution! Thank you for looking into it!